internal/cmd/skupper/link/link.go

```diff
@@ -34,16 +34,16 @@ func CmdLinkGenerateFactory(configuredPlatform common.Platform) *cobra.Command {
 
 	cmdLinkGenerateDesc := common.SkupperCmdDescription{
 		Use:   "generate",
-		Short: "Generate a new link resource in a yaml file",
-		Long: `Generate a new link resource with the data needed from the target site. The resultant
+		Short: "Generate a new link resource and save it as a YAML file, unless explicitly specified otherwise using the `--output` flag.",
```

Comment:
It is not behaving as the description says. It is always displaying a YAML output.

Reply:
"unless explicitly specified otherwise using the `--output` flag"?

Did not work with `-o json`?

Reply:
Yes, it works with json, but the description says: `Generate a new link resource and save it as a YAML file`.
Is it still a valid description?

Reply:
My bad, you are right, it is not generating any file. Will change that now.

Reply:
I have updated the description again, given that the -o flag is not available for non-kube.